### PR TITLE
is_group() function changed to attribute

### PR DESCRIPTION
**Pull request by meeuw**

### CODE REVIEW

**Comment by amjith** [comment]
Looks like the tests are failing. I'll take a look at it later today, but if you have time before that can you check the tests to see what's going on?

**Comment by meeuw** [comment]
this fixes the tests but I don't understand why is_group is a method during the tests...

File: mycli/packages/parseutils.py [hunk 1]
```diff
@@ -64,6 +64,9 @@ def last_word(text, include='alphanum_underscore'):
 # This code is borrowed from sqlparse example script.
 # <url>
 def is_subselect(parsed):
+    if isinstance(parsed, sqlparse.sql.Token):
+        if not parsed.is_group:
+            return False
     if not parsed.is_group():
```
Comment (tsroten) [inline mycli/packages/parseutils.py]: @meeuw Thanks for helping out with mycli! Is this call to `is_group()` still supposed to be in here?

Comment (meeuw) [inline mycli/packages/parseutils.py]: I'm not sure; I need to test it. If it's changed to is_group without () the tests fail... So somehow the unit tests are hitting this code.

**Comment by mdsrosa** [comment]
@meeuw Hey, thanks for helping out mycli! We really appreciate it! 
Did you have any chance to debug the code @tsroten mentioned?

Thanks again! 😄

**Pull request closed without merging**